app/models/persistent_volume.rb

```diff
@@ -1,4 +1,15 @@
 class PersistentVolume < ContainerVolume
   acts_as_miq_taggable
+  serialize :capacity
```

Comment:
We don't use hstore in the project and i didn't want to introduce it over here because of the tight schedule.
Changing to Hash.